model_cards/seyonec/ChemBERTa-zinc-base-v1/README.md

```diff
@@ -0,0 +1,14 @@
+# RoBERTA: Training a BERT-like transformer model for masked language modelling of chemical SMILES strings.
+
+Deep learning for chemistry and materials science remains a novel field with lots of potiential. However, the popularity of transfer learning based methods in areas such as NLP and computer vision have not yet been effectively developed in computational chemistry + machine learning. Using HuggingFace's suite of models and the ByteLeve tokenizer, we are able to train on a large corpus of 100k SMILES strings from a commonly known benchmark dataset, ZINC.
+
```

Comment:
ByteLeve => ByteLevel

Reply:
thanks for noticing that typo!

Reply:
I would include a example of usage @seyonechithrananda with ```transformers pipeline``` ```fill-mask```

Reply:
Will do!